guides/session.rst

```diff
@@ -1,76 +1,116 @@
 Controlling the Browser
 =======================
 
-Ok. Now we know how to create the browser driver to talk with a specific
-browser emulator. Although we can use drivers directly to call some actions
-on the emulator, Mink provides a better way - ``Session``:
+In Mink, the entry point to the browser is called the session. Think about
+it as being your browser window (some drivers even allow to deal with switching
+tabs).
+
+The first thing to do with your session is to start it. Nothing can be done
+with it before starting it.
```

Comment:
@weaverryan are you asking to replace the whole paragraph or only the first sentence ?